csm/conf/configure.py

```diff
@@ -137,7 +140,7 @@ def create(self):
 
     @staticmethod
     def _set_csm_endpoint():
-        Log.info("Setting csm endpoint in csm config")
+        Log.info("Setting csm endpoint in csm configuration")
```

Comment:
csm -> CSM
Setting csm endpoint in csm configuration -> Setting CSM endpoint in configuration.
Prepend all the info logs with the phase from where they are origination. 
Like Config: Setting CSM endpoint in configuration.

Reply:
updated